apps/app/src/app/pages/pages/components/modal-favorites/modal-favorites.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="favorites-container">
+  <!-- update title dependant of status -->
+  <button
+    class="btn btn-inverse btn-link"
+    title="{{
+      ((hasCurrentPageFavorite$() | async)
+        ? 'favorites.button.remove'
+        : 'favorites.button.add'
+      ) | translate
+    }}"
+    (click)="togglePageFavorite()"
+  >
+    <clr-icon
+      shape="star"
+      size="24"
+      [ngClass]="{ 'is-solid': hasCurrentPageFavorite$() | async }"
```

Comment:
Tu peux aussi mettre : `[class.is-solid]="hasCurrentPageFavorite$() | async`

Reply:
Ho, je connaissais pas cette syntax, j'apprécie ! 🚀